modules/standard/Help.chpl

```diff
@@ -20,6 +20,13 @@
 
 /* Functions for producing program help and usage.
 
+   .. note::
+
+    You must declare your main method to take an array of string arguments in
```

Comment:
When you say `main` you're talking about the function where your program starts its execution right?.

Unless I'm misunderstanding something I'd call it the `main` "procedure" or "function". I'd only use the word "method" only when referring to procs or functions that specifically appear inside classes or records.

Reply:
changed to "function" as that's what is used in the linked technote